src/main.js

```diff
@@ -16,4 +16,11 @@ var WebVRPolyfill = require('./webvr-polyfill.js');
 
 // Initialize a WebVRConfig just in case.
 window.WebVRConfig = window.WebVRConfig || {};
-new WebVRPolyfill();
+
+if (!window.WebVRConfig.DEFER_INITIALIZATION) {
+  new WebVRPolyfill();
```

Comment:
in fact, I would recommend opening a separate PR completely for this defer initialisation behaviour. I'll _defer_ to @borismus obviously as this is his library. but I think it'd make sense to just make breaking changes.

I'd suggest (à la [the `es6-promise` polyfill](https://github.com/stefanpenner/es6-promise#auto-polyfill)) to require the user to explicitly call a method on an exported module to polyfill.

so instead of having the `WebVRPolyfill` constructor automatically initialise the [polyfill](https://github.com/borismus/webvr-polyfill/blob/ca30e6d/src/webvr-polyfill.js#L28-L30),

so `WebVRPolyfill` would be exported and [these bits would be removed from the constructor](https://github.com/borismus/webvr-polyfill/blob/ca30e6d/src/webvr-polyfill.js#L28-L30) and [this line](https://github.com/borismus/webvr-polyfill/pull/33/files#diff-1c90ff38a08209f9ebd4d05d1e43358eL19) would get replaced with `module.exports = WebVRPolyfill` (and browserify would then attach that to `window`, handle AMD, CommonJS, etc.).